lib/STM.mli

```diff
@@ -126,6 +128,50 @@ sig
       This is helpful to model, e.g., a [remove] [cmd] that returns the removed element. *)
 end
 
+module type SpecExt =
+sig
+  (** Extended specification of a state machine.
+
+      This signature may be extended in the future with new specifications that
+      can be given defaults via {!SpecDefaults}. *)
+
+  include Spec
+
+  val wrap_cmd_seq : (unit -> 'a) -> 'a
+  (** [wrap_cmd_seq thunk] must call [thunk ()] and return or raise whatever
+      [thunk ()] returned or raised.  [wrap_cmd_seq] is used to wrap the
+      execution of the generated command sequences.  [wrap_cmd_seq] is useful,
+      e.g., to handle effects performed by blocking primitives. *)
```

Comment:
```suggestion
  (** [wrap_cmd_seq] is used to wrap the execution of the generated command
      sequences. [wrap_cmd_seq] is useful, e.g., to handle effects performed by
      blocking primitives. [wrap_cmd_seq thunk] must call [thunk ()] and return
      or raise whatever [thunk ()] returned or raised. *)
```
Nit: Moving the first sentence to the end improves the information flow I think:
Offering a general description first and finishing with requirement details.